configuration/device-tree.md

```diff
@@ -463,20 +539,19 @@ dtparam=foo=bar,level=42
 
 This will cause the loader to look for `overlays/acme-board.dtbo` in the firmware partition, which Raspbian mounts on `/boot`. It will then search for parameters `foo` and `level`, and assign the indicated values to them.
 
-The loader will also search for an attached HAT with a programmed EEPROM, and load the supporting overlay from there; this happens without any user intervention.
+The loader will also search for an attached HAT with a programmed EEPROM, and load the supporting overlay from there - either directly or by name from the "overlays" directory; this happens without any user intervention.
```

Comment:
Does the "or by name" imply that there's some kind of precedence if both are present?

Reply:
No, it's either/or. If the BLOB start 0xd00dfeed then it's an embedded overlay, otherwise we attempt to interpret it as an overlay name.

Reply:
None of that seems to be mentioned at https://github.com/raspberrypi/hats/blob/master/eeprom-format.md or https://github.com/raspberrypi/hats/blob/master/devicetree-guide.md ?

Reply:
I haven't finished the documentation spree yet - there's more joy to come.